README.md

```diff
@@ -70,12 +104,12 @@ In **PowerShell** or in a **Command Prompt** window:
 
 - **Windows**
 
-    - In PowerShell, upon executing `bundled_installer`, if you see an error with the message "execution of scripts is disabled on this system", set the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6) to "RemoteSigned" and then rerun `bundled_installer`.
+    - In PowerShell, if you encounter an error with the message "execution of scripts is disabled on this system", set the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6) to `RemoteSigned` and then rerun `bundled_installer`.
```

Comment:
To be precise, it should be "RemoteSigned" within backticks

Reply:
Got it, will update.